pixelblaze/pixelblaze.py

```diff
@@ -1490,7 +1541,7 @@ def sendPatternToRenderer(self, bytecode: bytes, controls: dict = {}):
         self.wsSendJson({"pause": False}, expectedResponse="ack")
         time.sleep(0.25)
 
-    def savePattern(self, *, previewImage: bytes, sourceCode: str, byteCode: bytes):
+    def savePattern(self, *, previewImage: bytes, sourceCode: str, byteCode: bytes, name: str, id: Union[str, None] = None):
```

Comment:
@zranger1 Tried a bunch of fiddling, but couldn't get this bad boi to twerk at all, nor find any examples. Realizing this changes the method signature, I could make a new one if you prefer?